fcrepo-http-api/src/main/java/org/fcrepo/http/api/url/HttpApiResources.java

```diff
@@ -123,8 +123,6 @@ private void addNodeStatements(final FedoraResource resource, final UriInfo uriI
 
                 //RDF the serialization
                 final Resource formatRDF = createResource(REPOSITORY_NAMESPACE + key);
-
-                model.add(formatRDF, RDFS_LABEL, key);
                 model.add(format, dcFormat, formatRDF);
```

Comment:
I know. I'm suggesting that if it doesn't matter for @osmandin then we should probably ignore it and not spend time investigating it, because it is going to go away anyway after HTTP conneg comes through.